src/main.js

```diff
@@ -297,12 +297,14 @@ const CodingManagerApplication = new Lang.Class({
         let active_window_xid = active_window ? active_window.get_xid() : 0;
         let current_window_xid = current_window ? current_window.get_xid() : 0;
 
-        // try to match transient windows
-        let transient_window = active_window.get_transient();
+        if (active_window != null) {
```

Comment:
So the early return when `active_window === null` happens when pressing reset button and then hovering over the notification. With the early return the window manager stays on the screen, and without the early return the window hides.

What's the preferred behavior?

I can see that the "early return" behavior is also in the eos-social app.